geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloFeatureReader.scala

```diff
@@ -98,6 +100,7 @@ class AccumuloFeatureReaderWithStats(query: Query,
       timings.time("next") + timings.time("hasNext"),
       count
     )
+    Monitoring.log(stat)
```

Comment:
would it make sense to have a 'master' stat writer implementation that used SPI (or something) to look up type-specific stat writers and delegated to them? that way we could have a single method/class to call, and could easily add other stat writing if desired.